server/shadow/shadow_encoder.c

```diff
@@ -131,8 +131,9 @@ static int shadow_encoder_uninit_grid(rdpShadowEncoder* encoder)
 
 static int shadow_encoder_init_rfx(rdpShadowEncoder* encoder)
 {
+	/* TODO: pass ThreadingFlags somehow */
```

Comment:
Maybe settings?

Reply:
explain please.

Reply:
`rdpShadowEncoder` contains `rdpSettings` for client and server instances.
You can add a setting and check that here if required.

Reply:
understood, will do